build/wix-install-wsl.ps1

```diff
@@ -29,5 +29,11 @@ function Install-Kernel {
     -Value "`"$Env:SystemRoot\system32\wsl.exe`" --update"
 }
 
+# Check if the Appx version of WSL has been installed.
+if (Get-AppxPackage -Name MicrosoftCorporationII.WindowsSubsystemForLinux) {
```

Comment:
Does this require the `-User` flag? to make sure that is fetching `.apx` for the current user?

Reply:
Hmm, I don't think so (though it's kind of confusing). When I tried the installer with this change, the installer finished quickly (rather than the usual minutes), and since `Get-WindowsOptionalFeature` usually takes forever I _hope_ it means it found the right user.

Either way, we don't have any values for `-User` if we do need it; it's unclear which user would eventually run Rancher Desktop…